ancestrymanager/ancestrymanager.go

```diff
@@ -324,6 +327,12 @@ func (m *manager) getProjectFromResource(d resources.TerraformResourceData, conf
 		if ok {
 			return res.(string), nil
 		}
+
+		res, ok = d.GetOk("parent")
```

Comment:
Sorry for not noticing this before; I think the reading of the `parent` field for `projects/XXXX` is currently not implemented the way we'd want. The logic here would only be called for Project / ProjectBillingInfo resources, which shouldn't have a `parent` field (so it'll have no effect.)

The logic in `getProjectFromSchema` is what's making this all work out right - technically, it would be better to get rid of `getProjectFromSchema` and include the logic directly in this function, but I am fine with leaving it there for now.

Reply:
So, the change request is to remove this section since it probably has no effect.